src/api/spec/features/webui/requests/submissions_spec.rb

```diff
@@ -103,5 +103,56 @@
         expect(page).to have_text('In state new')
       end
     end
+
+    describe 'when under the beta program' do
+      before { Flipper.enable(:request_show_redesign, receiver) }
```

Comment:
As an alternative you can:

```suggestion
    describe 'when under the beta program', beta: true do
```

To make this work, you should enable the feature by adding
`Flipper.enable(:request_show_redesign)`
to [spec/support/beta.rb](https://github.com/openSUSE/open-build-service/blob/e90a6f1a1c0ae800b140b3b4881523f1f7a6b12a/src/api/spec/support/beta.rb#L4) what we haven't done with `request_show_redesign` yet.